### PR TITLE
Address feedback on client-hints-01 draft

### PR DESCRIPTION
**Pull request by igrigorik**
WIP.

### CODE REVIEW

**Comment by igrigorik** [comment]
@mnot @reschke quick once over before I merge?

File: draft-ietf-httpbis-client-hints.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Client Hints does not supersede or replace the User-Agent header field. Existing
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in {{RFC2119}}.
 
-This document uses the Augmented Backus-Naur Form (ABNF) notation of {{RFC5234}} with the list rule extension defined in {{RFC7230}}, Appendix B. It includes by reference the DIGIT rule from {{RFC5234}}; the OWS, field-name and quoted-string rules from {{RFC7230}}; and the parameter rule from {{RFC7231}}.
+This document uses the Augmented Backus-Naur Form (ABNF) notation of {{RFC5234}} with the list rule extension defined in {{RFC7230}}, Appendix B. It includes by reference the DIGIT rule from {{RFC5234}}; the OWS and field-name rules from {{RFC7230}}.
```
Comment (reschke) [inline draft-ietf-httpbis-client-hints.md]: "It includes by reference the DIGIT rule ... and the OWS and field-name rules from ...."

File: draft-ietf-httpbis-client-hints.md [hunk 2]
```diff
@@ -143,12 +143,11 @@ For example:
   Accept-CH: DPR, Width, Viewport-Width, Downlink
 ~~~
 
-When a client receives Accept-CH, it SHOULD append the Client Hint header fields that match the advertised field-values. For example, based on Accept-CH example above, the client would append DPR, Width, Viewport-Width, and Downlink header fields to all subsequent requests.
-
+When a client receives Accept-CH, or if it is capable of processing the HTML response and finds an equivalent HTML meta element, it SHOULD append the Client-Hint header fields that match the advertised field-values to the header list of all subsequent requests. For example, based on Accept-CH example above, a user agent could append DPR, Width, Viewport-Width, and Downlink header fields to all subresource requests initiated by the page constructed from the response. Alternatively, a client may treat advertised support as a persistent origin preference and append same header fields on all future requests initiated to and by the resources associated with that origin.
```
Comment (reschke) [inline draft-ietf-httpbis-client-hints.md]: s/may treat/can treat/

File: draft-ietf-httpbis-client-hints.md [hunk 3]
```diff
@@ -244,10 +243,10 @@ If Downlink occurs in a message more than once, the minimum value should be used
 
 # The Save-Data Client Hint {#save-data}
 
-The "Save-Data" request header field is a token that indicates client's preference for reduced data usage, due to high transfer costs, slow connection speeds, or other reasons.
+The "Save-Data" request header field is one or more tokens that indicate client's preference for reduced data usage, due to high transfer costs, slow connection speeds, or other reasons.
 
 ~~~ abnf7230
```
Comment (reschke) [inline draft-ietf-httpbis-client-hints.md]: maybe "consists of"  instead of "is"?

**Comment by igrigorik** [comment]
@reschke thanks, updated, squashed.. merging.

**Pull request merged**